lib/stlink/bin/stlibp11_SAM.conf

```diff
@@ -0,0 +1,12 @@
+[Global]
+OutputFolder=C:\Users\Jos�Sim�es/STMicroelectronics/STM32CubeProgrammer/
```

Comment:
That may be adjusted somewhere? as won't work on anyone else machine

Reply:
Interesting.... that's coming from the tool folder, guess that's adjusted at install time.
Let me check if that can even be removed.